docs/debugging-testing/testing/testing_greentea.md

```diff
@@ -54,7 +54,7 @@ Tests can also be completely ignored by using the `.mbedignore` file described i
 
 #### Test names
 
-A test case is named by its position in your project's file structure. For instance, in the above example, a test case with the path `myproject/TESTS/test_group/test_case_1` would be named `tests-test_group-test_case_1`. The name is created by joining the directories that make up the path to the test case with a dash `-` character. This is a unique name to identify the test case. You will see this name throughout the build and testing process.
+A test case is named by its position in your project's file structure. For instance, in the above example, a test case with the path `mylibrary/tests/TESTS/test_group/test_case_1` would be named `tests-test_group-test_case_1`. The name is created by joining the directories that make up the path to the test case with a dash `-` character. This is a unique name to identify the test case. You will see this name throughout the build and testing process.
 
 ### Building tests
```

Comment:
Adding comment here for the rest of the file:
- Section Writing integration tests using host tests: update the location of host tests to `mbed-os/TESTS/host_tests` and sy that this directory should be created if it does not exist.
- Output extracts should be updated with new directory structure